src/groovy/org/transmartproject/db/util/SimpleScrollableResults.groovy

```diff
@@ -0,0 +1,55 @@
+package org.transmartproject.db.util
+
+import org.hibernate.ScrollableResults
+
+class SimpleScrollableResults<T> implements Closeable, Iterable<T> {
```

Comment:
The naming is off here. Don't call it `SimpleScrollableResults` if it doesn't implement the `ScrollableResults` interface; it's confusing. Something like `SimpleScrollableResultsWrappingIterator`, while longer, is more descriptive.